esrally/driver/driver.py

```diff
@@ -1432,7 +1432,11 @@ async def execute_single(runner, es, params, on_error):
         elif e.info:
             request_meta_data["error-description"] = "%s (%s)" % (e.error, e.info)
         else:
-            request_meta_data["error-description"] = e.error
+            if isinstance(e.error, bytes):
+                error_description = e.error.decode("utf-8")
+            else:
+                error_description = str(e.error)
```

Comment:
Would it be soo bad to leave as None given thats the only other viable type? For safety this probably makes sense.

Reply:
As discussed elsewhere I'd rather be a bit more cautious here and always enforce conversion to a string.